openedx/features/assessment/helpers.py

```diff
@@ -29,13 +29,18 @@ def can_auto_score_ora(enrollment, course, block, index_chapter):
     anonymous_user = get_anonymous_user(enrollment.user, course.id)
     if not anonymous_user:
         return False
-    response_submission = Submission.objects.filter(
+    response_submissions = Submission.objects.filter(
         student_item__student_id=anonymous_user.anonymous_user_id,
         student_item__item_id=block
-    ).order_by('submitted_at').first()
+    ).order_by('submitted_at')
+    log.info('Multiple submissions found having ids {ids} in can_auto_score_ora'.format(
+        ids=[submission.id for submission in response_submissions]
```

Comment:
How about to print the ids nicely. Would be great if we can make this change in other places as well
`', '.join([submission.id for submission in response_submissions])`